gamelog.txt

```diff
@@ -0,0 +1,11 @@
+q guessed the sequence ["g", "y", "r", "y"] in 2 rounds within 27 seconds
```

Comment:
Add this to the git ignore and remove it from cache. `git rm --cached gamelog.txt`